framework/i18n/Formatter.php

```diff
@@ -787,6 +787,54 @@ public function asRelativeTime($value, $referenceTime = null)
         }
     }
 
+    /**
+     * Formats the value as a time interval.
+     * @param DateInterval|string|integer $value the value to be formatted, in ISO8601 duration format or number of seconds.
+     * @return string the formatted result.
+     * @since 2.0.7
+     */
+    public function asDuration($value)
```

Comment:
Oh, you should comment it on line 835 containing the return statement. I thought you mean the dash used in `str_replace()` calls.